kpatch-build/create-diff-object.c

```diff
@@ -760,11 +760,45 @@ static void kpatch_compare_symbols(struct list_head *symlist)
 	}
 }
 
+#define CORRELATE_ELEMENTS(e1, e2)				\
+do {								\
+	e1->twin = e2;						\
+	e2->twin = e1;						\
+	/* set initial status, might change */			\
+	e1->status = e2->status = SAME;				\
+	if (strcmp(e1->name, e2->name)) {			\
+		/* Rename mangled element */			\
+		log_debug("renaming %s to %s\n",		\
```

Comment:
Would be nice if we could somehow differentiate between sections and symbols here.

Reply:
So, have two separate static functions?

Reply:
Well every way of doing this seems to be as inconvenient as separate functions... But since those are bundled together maybe a `log_debug` at the beginning of `kpatch_correlate_symbols` and `kpatch_correlate_sections` will be enough.

Reply:
Oh, so you meant differentiate the message printed?

Yes, that makes sense, I'll do that in the next version.